### ISSUE TITLE
Linux build

### ISSUE THREAD

**Issue by dromer** [publicsamples/Quetzalcoatl#9]
Since HISE supports Linux, please add this to your builds! :)

**Comment by RickTaylor79** [comment]
It's definitely on the cards, there might be a bit of a wait though as my x86 machine died. Happy to help you though the build process if you wanted to roll your own.

**Comment by RickTaylor79** [comment]
Okay, finally got round to a Linux build. It's been tested minimally in Reaper on a 2012(!) iMac. Seems to work & okay. 

https://github.com/publicsamples/Quetzalcoatl/releases/tag/0.5.3

**Comment by dromer** [comment]
On which OS did you try it? I can sort of load it in Carla, but I can't get any audio.
Bitwig and carla-single refuse though:

```
dreamer@auditorium:~$ carla-single vst3 .vst3/Quetzalcoatl.vst3/
['/usr/lib/carla/carla-bridge-native', 'vst3', '.vst3/Quetzalcoatl.vst3/', '(none)', '0']
libjack.so.0 loaded successfully!
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  20 (X_GetProperty)
  Resource id in failed request:  0x0
  Serial number of failed request:  54
  Current serial number in output stream:  54
[carla] Carla assertion failure: "engine == nullptr" in file ../backend/CarlaHostImpl.hpp, line 87
[carla] Carla assertion failure: "! lib.canDelete" in file ../utils/CarlaLibCounter.hpp, line 46
```

**Comment by RickTaylor79** [comment]
It was compiled using Ubuntu 20.04. I've just tried it in Bitwig and it works as expected (couldn't get Carla to detect any VST3 plugins).

Did you install the sample content? It won't make a sound without out this:

https://github.com/publicsamples/Quetzalcoatl/wiki/Installing-Sample-Content

Building it yourself should be pretty easy, I've covered the basics here:

https://github.com/publicsamples/Quetzalcoatl/wiki/Build-it-yourself
